File: Week 1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{

    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    for (int i = 1; i <= height; i++)
    {
        for (int j = height - i; j >= 1; j--)
        {
            printf(" ");
        }
        for (int x = 1; x <= i; x++)
        {
            printf("#");
        }
        printf("  ");
        for (int y = 1; y <= i; y++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: Week 6/sentimental-mario-more/mario.py
from cs50 import get_int

h = 0
while h < 1 or h > 8:
    h = get_int("Height: ")

for i in range(h):
    for j in range(h - i - 1):
        print(" ", end="")
    for x in range(i + 1):
        print("#", end="")
    print("  ", end="")
    for y in range(i + 1):
        print("#", end="")
    print()
